apps/nextra/package.json

```diff
@@ -12,7 +12,9 @@
     "prebuild": "node nextra-remote-filepaths/fetch.js",
     "predev": "pnpm prebuild",
     "start": "next start -p 3030",
-    "types:check": "tsc --noEmit"
+    "types:check": "tsc --noEmit",
+    "fmt": "prettier --write '**/*.(tsx|ts|jsx|js|json|md|html)'",
+    "lint": "prettier --check '**/*.(tsx|ts|jsx|js|json|md|html)'"
```

Comment:
Should this be `mdx`?

Reply:
Ah yeah, I should add `mdx` there. I just ripped this straight from the docusaurus package.json

Reply:
I just ran the format script on all of our `mdx` files and it made some strange formatting changes that broke a bunch of the files. I guess Prettier support for MDX isn't there yet and we'll have to go without it for now.